lib/files/index.js

```diff
@@ -0,0 +1,120 @@
+const { mkdir, readdir, rm, rename, stat, writeFile } = require('fs/promises')
+const { existsSync }  = require('fs')
+const { dirname, join, normalize } = require('path')
+const multer = require('multer')
+
+const  filesInterface = (app, settings)  => {
+    const BASE_PATH = join(settings.rootDir, settings.userDir, 'storage')
+    // not sure about this yet, need to check how file storage works
+    const storage = multer.memoryStorage()
+    const upload = multer({storage: storage})
+
+    const regex = /^\/flowforge\/files\/_\/([\.a-zA-Z0-9\/\-_ ]*)$/
+
+    const listDirectory = async (path = '') => {
+        const fullPath = normalize(join(BASE_PATH, path))
+        if (fullPath.startsWith(BASE_PATH)) {
+            const files = await readdir(fullPath, {withFileTypes: true})
+            const response = {
+                meta: {
+                    next_cursor: ''
+                },
```

Comment:
```suggestion
                meta: { },
```
We only need to return the `next_cursor` if there are more results to return. For first iteration, we can just always return the whole listing.